lib/main.dart

```diff
@@ -59,6 +59,7 @@ Future<void> main() async {
   }
   void appRunner() => runApp(MultiProvider(
           providers: [
+            Provider<PlayerModel>(create: (_) => PlayerModel()..load()),
```

Comment:
This bit in particular... I'm not sure if this is the right way to get the data in there.

Reply:
Pretty sure this is redundant with the line right below?

Reply:
Oh it does look that way! Just removed it.